db/seeds.rb

```diff
@@ -24,5 +24,13 @@
   else
     flag = false
   end
-  patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  pregnancy = patient.pregnancies.create({last_menstrual_period_time: DateTime.new(2016,1,1), urgent_flag: flag})
+  5.times do
+    pregnancy.calls.create({status: 'Left voicemail', creating_user_id: user.id})
+  end
+  if patient.name == 'Patient 0'
+    10.times do
+      pregnancy.calls.create({status: 'Reached patient', creating_user_id: user.id})
```

Comment:
I'm currently getting an error when running `rake db:seed` with this: 

```
$ rake db:seed
rake aborted!
Mongoid::Errors::UnknownAttribute: 
message:
  Attempted to set a value for 'creating_user_id' which is not allowed on the model Call.
```

I think we can resolve if we change both the `creating_user_id`s to `created_by`s, like so: 

`pregnancy.calls.create({status: 'Left voicemail', created_by: user.id})`

@mchelen thoughts?